compiler/testData/codegen/bytecodeText/forLoop/forInIndices/forInCollectionIndices.kt

```diff
@@ -12,6 +11,16 @@ fun test() {
 // 0 getFirst
 // 0 getLast
 
+// JVM non-IR uses while.
+// JVM IR uses if + do-while.
+
+// JVM_TEMPLATES
 // 0 IF_ICMPGT
```

Comment:
These two lines (`// 0 IF_...`) can be removed; they are made redundant by the `// 1 IF`

Reply:
Thanks. Done.